TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py

```diff
@@ -317,21 +320,26 @@ def copyPythonScript(self):
 
             # OGS
             if "constant_contributions.npy" in line:
-                line = line.replace("constant_contributions.npy",
-                                    constants_filename)
+                line = line.replace(line,
+                                    "constant_contributions = np.load('" +
```

Comment:
For it to run with Windows, all paths need to be interpreted as raw string -> add `r` before each path.

```suggestion
                                    "constant_contributions = np.load(r'" +
```

Reply:
see commit... does it still work on linux?

Reply:
Works perfectly fine.
Thanks for the commit!
I tried to update #158 accordingly. Can you please test?